core/API/Request.php

```diff
@@ -261,7 +262,12 @@ public function process()
             // call the method
             $returnedValue = Proxy::getInstance()->call($apiClassName, $method, $this->request);
 
-            $toReturn = $response->getResponse($returnedValue, $module, $method);
+            // get the response with the request query parameters loaded, since DataTablePost processor will use the Report
```

Comment:
@diosmosis does it make sense to move `$returnedValue = Proxy::getInstance()->call($apiClassName, $method, $this->request);` into this `Context::executeWithQueryParameters` below and remove it from the `call` method? The call method isn't API so we can change it.

Reply:
Some code does call `Proxy::getInstance()->call()` directly, eg, `DataTableManipulator`s so I think it could break things.